src/content/ui-api/monitoring/index.md

```diff
@@ -0,0 +1,66 @@
+---
+linkTitle: Monitoring
+title: Monitoring and dashboards
+description: Giant Swarm runs a monitoring stack in each installation, comprising Prometheus, Grafana, and more. Learn how to get access to the data we collect.
+last_review_date: 2021-06-28
+user_questions:
+  - How can I access Grafana for my installation?
+  - How can I access metrics from my installation?
+  - Where can I find workload cluster metrics?
+  - Which dashboards do I have access to?
+  - Which metrics are gathered in my installation?
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-atlas
+---
+
+# Monitoring and dashboards
+
+Giant Swarm runs a monitoring stack in each installation, based on [Prometheus](https://prometheus.io/), collecting data from most management cluster components, as well as many workload cluster metrics.
+
+For accessing these metrics, both for you as customers and for our staff we provide a [Grafana](https://grafana.com/) instance per installation.
+
+![Home dashboard](home-dashboard.png)
+
+We aim to provide a useful set of dashboards for you to quickly explore all metrics relevant to you, giving you access to the same information that we use for operations, support, and billing.
+
+## Accessing dashboards
+
+If you have access to the Giant Swarm [web interface]({{< relref "/ui-api/web" >}}), the easiest way to get to Grafana is to click the link **Monitoring** in the top menu when logged in.
```

Comment:
random brain wave: weren't you working on something that would show people their base domain URLs or something? it'd be really cool to directly link from docs to their installation's grafana

Reply:
Yes, but it's far from done.